datastore/longhorn.go

```diff
@@ -1403,6 +1403,24 @@ func (s *DataStore) CheckEngineImageReadyOnAllVolumeReplicas(image, volumeName,
 	return s.CheckEngineImageReadiness(image, nodes...)
 }
 
+func (s *DataStore) IsDefaultEngineInstanceManagerAvailability(nodeID string) bool {
+	return s.CheckDefaultEngineInstanceManagerAvailability(nodeID) == nil
+}
+
+func (s *DataStore) CheckDefaultEngineInstanceManagerAvailability(nodeID string) error {
```

Comment:
Suggest merging this to `IsDefaultEngineInstanceManagerAvailability` if `CheckDefaultEngineInstanceManagerAvailability` is not required to export and used by others.

Reply:
Seems we can only filter by label selector. 
https://github.com/longhorn/longhorn-manager/blob/master/k8s/pkg/client/listers/longhorn/v1beta2/backup.go#L63